src/vl-core.js

```diff
@@ -226,6 +226,16 @@ export const NativeVlElement = (SuperClass) => {
      * @extends VlElement
      */
     class NativeVlElement extends VlElement(SuperClass) {
+        /**
+         * NativeVlElement constructor. Deze geeft geen html mee zoals bij {VlElement},
+         * aangezien {NativeVlElement}en geen shadow dom mogen aanmaken.
```

Comment:
Mogen/kunnen.

Reply:
Een aantal inderdaad niet, maar sommige zouden dat wel kunnen, bvb bij een extend van een div element.

Reply:
ben niet echt mee wat een constructor oplost aan styling problemen?

Reply:
Zie ook UIG-518 : styling problemen in combinatie met een fout in de console. Doordat het probleem zich voordoet in de constructor van het element, wordt de html element wel deels geladen, maar zonder de style. Dit lost het op omdat anders de document-register library een constructor aanmaakt met een argument, wat voordien de constructor van de superclass oproept, met dus html voor in de shadow dom te duwen (zie constructor van VlElement). Waarom die library dit zo doet heb ik niet in detail bekeken, maar kan nog wel verder onderzocht worden indien je dat nuttig zou vinden (bvb door dat het probleem misschien bij die library kan liggen om dit dan te melden daar).